### PR TITLE
host-HA vs VM-HA support

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

File: source/adminguide/reliability.rst [hunk 1]
```diff
@@ -186,6 +186,11 @@ that you want to dedicate to HA-enabled VMs.
 HA-Enabled Hosts
 ----------------
 
+.. note::
+   This feature is only applicable to KVM clusters. It is not supported
```
Comment (harikrishna-patnala) [inline source/adminguide/reliability.rst:190]:
```suggestion
   This feature is only applicable to KVM clusters. It is not supported
   on for instance VMware or Xen. For those hypervisor types, the Host HA
   is left to the VMware-cluster or Xen-pool respectively.
```

Comment (DaanHoogland) [reply source/adminguide/reliability.rst:190]: never mind got it

**Comment by harikrishna-patnala** [comment]
@blueorangutan docbuild

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

image

and

image

**Pull request merged**